bin/juxtaposer/formatter/stdout/stdout.go

```diff
@@ -14,22 +15,22 @@ func NewFormatter(options formatter_api.FormatterOptions) (formatter_api.OutputF
 	return &StdoutFormatter{}, nil
 }
 
-func (formatter *StdoutFormatter) ProcessResults(backendNames []string, aggregatedTimings map[string]formatter_api.BackendTiming, baselineThresholdMaxPercent int) error {
+func (formatter *StdoutFormatter) ProcessResults(backendNames []string, aggregatedTimings map[string]timing.BackendTiming, baselineThresholdMaxPercent int) error {
 	fields := []map[string]string{
 		map[string]string{"name": "Name", "nameFormat": "%-30s", "valueFormat": "%-30s"},
-		map[string]string{"name": "Min", "nameFormat": "%12s", "valueFormat": "%12v"},
-		map[string]string{"name": "Max", "nameFormat": "%12s", "valueFormat": "%12v"},
-		map[string]string{"name": "Avg", "nameFormat": "%12s", "valueFormat": "%12v"},
+		map[string]string{"name": "Min", "nameFormat": "%13s", "valueFormat": "%13v"},
+		map[string]string{"name": "Max", "nameFormat": "%13s", "valueFormat": "%13v"},
+		map[string]string{"name": "Avg", "nameFormat": "%13s", "valueFormat": "%13v"},
 		map[string]string{"name": "90% Lower %", "nameFormat": "%12s", "valueFormat": "%12.2f"},
 		map[string]string{"name": "90% Upper %", "nameFormat": "%12s", "valueFormat": "%12.2f"},
-		map[string]string{"name": "Total", "nameFormat": "%13s", "valueFormat": "%13v"},
-		map[string]string{"name": "Rounds", "nameFormat": "%7s", "valueFormat": "%7d"},
-		map[string]string{"name": "Errors", "nameFormat": "%7s", "valueFormat": "%7d"},
+		map[string]string{"name": "Total", "nameFormat": "%18s", "valueFormat": "%18v"},
+		map[string]string{"name": "Rounds", "nameFormat": "%9s", "valueFormat": "%9d"},
+		map[string]string{"name": "Errors", "nameFormat": "%9s", "valueFormat": "%9d"},
 		map[string]string{"name": "Succ %", "nameFormat": "%9s", "valueFormat": "%9.2f"},
 		map[string]string{"name": "Thresh %", "nameFormat": "%9s", "valueFormat": "%9.2f"},
 	}
 
-	dividerString := strings.Repeat("-", 146)
+	dividerString := strings.Repeat("-", 158)
```

Comment:
Similar to my comment about rolling your own stats functions, I don't think we should be rolling your own ascii table formatter.  This is stock utility code.  Eg, 

https://github.com/olekukonko/tablewriter

would probably work well.

This change doesn't need to be made now.

Reply:
Yup, we can figure out how to do the numbers after we also include the rolling avg for the baseline too.

Reply:
I have some architectural ideas around the formatter, but those will def wait for the full review.  Just planting a flag.